cs/test/internal/Internal.csproj

```diff
@@ -33,15 +24,11 @@
     <Reference Include="Bond">
```

Comment:
Could these Bond references be turned into `ProjectReference`?
Same comment for the other places where we have explicit `HintPath` references still.

Reply:
Tests projects I have not yet cleaned up, since I've been actively moving things around for CI.